src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -2846,6 +2891,12 @@ private static void FormatTrackedParameters(RequestDelegateFactoryContext factor
         }
     }
 
+    private static string[] SplitAndTrim(string[] values)
+    {
+        var result = values.SelectMany(v => v.Split(',').Select(s => s.Trim())).ToArray();
+        return result;
+    }
```

Comment:
This looks really inefficient. This needs to allocate as a little memory as possible. That means this code needs to be quite a bit more complex.

Reply:
Thank you for the feedback! I have attempted to increase the performance and memory efficiency of the `SplitAndTrim` method.